dev-python/jinja/jinja-2.11.3.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
```

Comment:
You could try putting an older version of jinja that is compatible with python3.6 after #212 gets merged. So you wouldn't need to modify ebuilds in portage-stable.